Makefile

```diff
@@ -86,8 +86,9 @@ $(BIN): $(GOSRC)
 .PHONY: data
 data: $(DATA)
 
+URFAVECLI_PATH := $(shell ./urfave_cli_path.sh)
 %.bash: $(GOSRC)
-	go run $(BUILDFLAGS) -ldflags "$(LDFLAGS)" . --generate-bash-completion > $@
+	cp $(URFAVECLI_PATH)/autocomplete/bash_autocomplete $@
```

Comment:
Is it necessary to use the latest version of the bash script from `urfave` Go package? The script is very simple:

```bash
#! /bin/bash

: ${PROG:=$(basename ${BASH_SOURCE})}

_cli_bash_autocomplete() {
  if [[ "${COMP_WORDS[0]}" != "source" ]]; then
    local cur opts base
    COMPREPLY=()
    cur="${COMP_WORDS[COMP_CWORD]}"
    if [[ "$cur" == "-"* ]]; then
      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} ${cur} --generate-bash-completion )
    else
      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
    fi
    COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
    return 0
  fi
}

complete -o bashdefault -o default -o nospace -F _cli_bash_autocomplete $PROG
unset PROG
```

Couldn't we just copy the script to our repository? I'm not enforcing. I'm just asking.

Reply:
Yes, I've later discussed as better option to have the script included directly exactly because it is simple and probably will not be changed in future.